collect_app/src/main/java/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -291,40 +285,6 @@ public QuestionDetails getQuestionDetails() {
         return questionDetails;
     }
 
-    // http://code.google.com/p/android/issues/detail?id=8488
-    private void recycleDrawablesRecursive(ViewGroup viewGroup, List<ImageView> images) {
```

Comment:
All of these is removed because it caused crashes since Glide using caching. I could disable caching in Glide and then keep this code but it's not recommended and I believe it's a very old code that we no longer need because we have more powerful devices and also glide should deal with it better.

Reply:
This seems reasonable to me. Am I understanding correctly that the Glide cache will be some kind of singleton so on activity rotation all images should be able to come from some cache layer?

Reply:
Yes I'm not sure about rotating and would need to double check but it uses caching for example when we navigate between questions so probably during rotating as well.